docs/Analog.md

```diff
@@ -1,32 +1,95 @@
+
+1. Accept truly differential, bipolar, ±10V (max) signals at each of the differential analog inputs.
+2. Synchronously digitize all differential analog inputs at the same time (no multiplexing).
+3. Operate ADC fast enough such that the sampling can be aligned to the PWM carrier waveform (>100ksps throughput on all channels).
+4. Ensure there are no possible ground loops by using high impedance inputs for the differential voltages.
+5. Ensure a high common-mode rejection ratio (CMRR) for the differential inputs.
+6. Ensure the analog signal chain has minimal noise.
+
+As mentioned above, the analog input to the AMDC should be bipolar differential with the voltage within ±10V. Then, the analog front-end converts this into a single-ended pseudo-differential signal which is needed by the analog-to-digital converter (ADC). More information on the ADC input types and their definitions can be found [here](https://www.analog.com/media/en/technical-documentation/product-selector-card/2PB_sarinputtypesfb.pdf).
 
 ## Block Diagram / External Connections
 
-Pictorial representation of analog system, with blocks and descriptions of what they do.
+<img src="images/amdc-analog.svg" />
 
-## Schematics
+### Analog connectors
+The analog signal chain of the AMDC can receive up to 8 bipolar differential analog inputs (16 signals in total – 8 positive and 8 negative). There are 4 analog connectors (modular jacks) each receiving up to 2 differential inputs. There are also 4 ESD protection chips to protect the sensitive electronics from the damage due to ESD.
 
-Discussion related to schematics for analog subsystem:
-- Related screenshots
-- Hierarchy
+A table of the pin mappings for the analog connectors is shown below:
 
-## PCB Layout
+| Pin number | `Analog connector 1` | `Analog connector 2` | `Analog connector 3` | `Analog connector 4` |
+|------------|--------|-----|-----|-----|
+| 1 | `Analog P1` | `Analog P3` | `Analog P5` | `Analog P7` |
+| 2 | `Analog N1` | `Analog N3` | `Analog N5` | `Analog N7` |
+| 3 | +15V | +15V | +15V | +15V |
+| 4 | `GND` | `GND` | `GND` | `GND` |
+| 5 | -15V | -15V | -15V | -15V |
+| 6 | `GND` | `GND` | `GND` | `GND` |
+| 7 | `Analog P2` | `Analog P4` | `Analog P6` | `Analog P8` |
+| 8 | `Analog N2` | `Analog N4` | `Analog N6` | `Analog N8` |
+
+The location of these 4 analog connectors in the AMDC (stacked as 2 x 2) is shown below:
+
+<img src="images/amdc-analog-input-highlighted.svg" />
+
+In this picture, the top left connector (when looking from the right side) corresponds to the `Analog connector 1` in the block diagram. The top right connector is the `Analog connector 2`, the bottom left is the `Analog connector 3`, and the bottom right is the `Analog connector 4`.
+
+More detailed information on the analog connectors can be found in the [datasheet](https://media.digikey.com/pdf/Data%20Sheets/Amphenol%20PDFs/RJSAE_Brochure.pdf).

```

Comment:
@Anvar-Khamitov Try to put connector numbers `1` - `4` right in this SVG file so that we can immediately see the connector numbering.

Reply:
@Anvar-Khamitov This doesn't look very professional...

Here's an idea: what if you draw small red circles beside each connector and inside the circles you have a black or white number. You'll have to make the font size for the numbers smaller than it currently is.

See what you can do to make these numbers look more readable and to not fully obscure each port.

Reply:
@elsevers How about now?

Reply:
It is looking better, but improvement is still needed. I actually meant for you to have a filled in circle that your number appears on top of. I think you should use one of the colors from your block diagram

Reply:
@elsevers Like now?

Reply:
Looks good!